backend/internal/controller/server/msg_handler.go

```diff
@@ -66,3 +70,43 @@ func (s server) createResponse(reader io.Reader) ([]byte, error) {
 func (s server) ResetScoreHandler(w http.ResponseWriter, r *http.Request) {
 	s.game.ResetScore()
 }
+
+// ManipulateScoreHandler is a handler for manipulation of the score.
+// Incoming URL should be in the format: '/goal?action=[add/sub]&team=[1/2]'.
+// Team ID 1 stands for white and 2 for blue.
+func (s server) ManipulateScoreHandler(w http.ResponseWriter, r *http.Request) {
+
+	team := r.URL.Query().Get(attributeTeam)
+
+	teamID, err := strconv.Atoi(team)
+	if err != nil || !isValidTeamID(teamID) {
+		writeHTTPError(w, http.StatusBadRequest, "Team ID has to be a number either 1 or 2")
+		return
+	}
+
+	switch action := r.URL.Query().Get(attributeAction); action {
+	case "add":
+		s.game.AddGoal(teamID)
+	case "sub":
+		// TODO: s.game.SubGoal(teamID)
+		if err = writeHTTPError(w, http.StatusBadRequest, "Action not implemented yet"); err != nil {
+			log.Println(err)
+		}
+	default:
+		if err = writeHTTPError(w, http.StatusBadRequest, "Wrong action"); err != nil {
+			log.Println(err)
+		}
+
+	}
+}
+
+func writeHTTPError(w http.ResponseWriter, header int, msg string) error {
+	log.Println("Error handling request: ", msg)
+	w.WriteHeader(header)
+	_, err := w.Write([]byte(msg))
+	return err
```

Comment:
You could just log this error inside `handleError`, that'd make it more pleasant to use